src/elements/content-uploader/__tests__/Footer.test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import { render, screen } from '../../../test-utils/testing-library';
+import Footer from '../Footer';
```

Comment:
Move this below  the `noop` import

Reply:
So this is a edge case, if I move the footer import above the test-utils/ import, all the test fail because, in the testing import its unmocking the intl stuff, until we can completely unmock react-intl we cant move the import above it.

If we have to move it above the testing import I have to add 
`jest.unmock('react-intl');` to the test file.

Which one do you prefer? leaving as is or move and add the unmock directly to the test file?

Reply:
Oh interesting, I have no preference, I think leaving as is is okay then